c/include/nnstreamer-single.h

```diff
@@ -259,6 +265,82 @@ int ml_single_set_property (ml_single_h single, const char *name, const char *va
  */
 int ml_single_get_property (ml_single_h single, const char *name, char **value);
 
+/*************
+ * ML OPTION *
+ *************/
+
+/**
+ * @brief Create ml-option instance.
+ * @details Some detail information should be written here.
+ * @since_tizen 7.0
+ * @param[out] option Newly created option handle is returned.
+ * @return @c 0 on success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_INVALID_PARAMETER Fail. The parameter is invalid.
+ * @retval #ML_ERROR_OUT_OF_MEMORY Failed to allocate required memory.
+ */
+int ml_option_create (ml_option_h *option);
+
+/**
+ * @brief Destroy ml-option instance.
+ * @details Some detail information should be written here.
+ * @since_tizen 7.0
+ * @param[in] option The option handle to be destroyed.
+ * @return @c 0 on success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_INVALID_PARAMETER Fail. The parameter is invalid.
+ */
+int ml_option_destroy (ml_option_h option);
+
+/**
+ * @brief Set ml-option key-value.
+ * @details key is string and value is void pointer. Note that the value should be valid during single task and freed after using. shortly, the caller should manage the value pointer. key is copied to hash tables, so doesn't need to free.
+ * If some options are changed or there are newly added options, please modify below description.
+ * The list of key-values are:
+ * key (char *) ||| value (expected type (pointer))
+ * "fw_version" ||| ("1" | "2") -> only applied to nnfw tensorflow-lite.
```

Comment:
More explanation for fw_version required.
Users won't be able to understand what is fw_version here.